### PR TITLE
Latest api derive

### PR DESCRIPTION
**Pull request by cinlk**

### CODE REVIEW

File: packages/api-derive/src/darwiniaStaking/power.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  return api.query.staking.bonded(accountId).pipe(
+    switchMap((account): Observable<PowerOf> =>
+      account.isNone
+        ? of({ power: new BN('0') })
+        : api.queryMulti<[u128, u128, Option<DarwiniaStakingStructsStakingLedger>]>([
+          api.query.staking.ktonPool,
+          api.query.staking.ringPool,
+          [api.query.staking.ledger, account.unwrap()]
+        ]).pipe(
+          map(([ktonTatal, ringTatal, ledger]): PowerOf => {
```
Comment (acgxv) [inline packages/api-derive/src/darwiniaStaking/power.ts]: tatal?

File: packages/api-derive/src/darwiniaStaking/power.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import { DeriveApi } from '@polkadot/api-derive/types';
+import { memo } from '@polkadot/api-derive/util';
+import { AccountId } from '@polkadot/types/interfaces';
+import { DarwiniaStakingStructsStakingLedger } from '@polkadot/types/lookup';
+import { Option, u128 } from '@polkadot/types-codec';
+import { BN } from '@polkadot/util';
+
+import { PowerOf } from './types';
+
+const ration = 500000000;
```
Comment (acgxv) [inline packages/api-derive/src/darwiniaStaking/power.ts]: does ts support 500_000_000?

File: packages/api-derive/src/bundle.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import type { PowerOf } from './darwiniaStaking/types';
+import type { DeriveMarketFee } from './feeMarket/types';
+
+import { AccountId, LaneId, MessageNonce } from '@polkadot/types/interfaces';
+import { AnyNumber, ITuple } from '@polkadot/types-codec/types';
+
+declare module '@polkadot/api-derive/derive'{
+
+  export interface ExactDerive {
+
```
Comment (sxlwar) [inline packages/api-derive/src/bundle.ts]: format code pls

**Review by sxlwar — CHANGES_REQUESTED** [review]

File: packages/api-derive/src/darwiniaStaking/power.ts (hunk 1 above)
Comment (cinlk) [reply packages/api-derive/src/darwiniaStaking/power.ts]: > tatal?

well, it's total

File: packages/api-derive/src/darwiniaStaking/power.ts (hunk 2 above)
Comment (cinlk) [reply packages/api-derive/src/darwiniaStaking/power.ts]: > does ts support 500_000_000?

change to 500_000_000

**Review by acgxv — APPROVED** [review]

**Review by sxlwar — APPROVED** [review]

**Pull request merged**